### PR TITLE
Add new actions: list-paths hosts-for-path

### PR DESCRIPTION
**Pull request by danpersa**

### CODE REVIEW

File: src/instaskip/actions/paths.clj [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                     {:team team}
+                     ; =>
+                     (ik/get-paths {:owned-by-team team} innkeeper-config)
+
+                     :else
+                     ; =>
+                     (ik/get-paths innkeeper-config))
+
+        paths-with-hosts (->> paths
+                              (map (fn [{:keys [id uri]}]
```
Comment (mo-gr) [inline src/instaskip/actions/paths.clj:15]: could be (select-keys paths [:id :uri])

File: src/instaskip/actions/routes.clj [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+(defn list-routes [filters innkeeper-config]
+
+  (match filters
+         {:team team}
+         ; =>
+         (ik/get-paths {:owned-by-team team} innkeeper-config)
+
+         :else
+         ; =>
```
Comment (mo-gr) [inline src/instaskip/actions/routes.clj]: why the `; =>`?

**Comment by mo-gr** [comment]
👍

File: src/instaskip/actions/routes.clj (hunk 2 above)
Comment (danpersa) [inline src/instaskip/actions/routes.clj]: I stole it from this [article](http://matt.might.net/articles/parsing-with-derivatives/). I guess the intention is to make the code more readable for newbies. Do you think I should remove it? Or do you think it helps? :)

File: src/instaskip/actions/paths.clj (hunk 1 above)
Comment (danpersa) [inline src/instaskip/actions/paths.clj:15]: This code is doing destructuring. `id` and `uri` get bound to the map for the specified keys. `select-keys` would only remove the unwanted keys from the map.

**Pull request merged**